roles/dosomething.phoenix/templates/nginx/app.j2

```diff
@@ -30,6 +30,7 @@ server {
   location / {
     # This is cool because no php is touched for static content
     try_files $uri @rewrite;
+    error_page 404 /leftathetrafficlight.php;
```

Comment:
Ah, I see in in #5298. Please add the comment clarifying that this is intentionally.